### PR TITLE
Create Guessing_game.py

### PR DESCRIPTION
**Pull request by Dark-Atom**
This is small game made with python,In this game you have to guess any number and that print a status of your guess. For more information run program and please, read instruction.

### CODE REVIEW

**Comment by poyea** [comment]
Please put this in an appropriate folder, with appropriate name (if you need a new folder).

File: python games/Guessing_game.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        b=random.randint(1,100)
+        while(True):
+            if(b>a):
+                print("your guess is too low")
+                a=int(input("try again:"))
+                if a==0:
+                    print("Thanks for playing!!")
+                    break
+            elif(b<a):
+                print("you guess is too high")
```
Comment (havanagrawal) [inline python games/Guessing_game.py]: *your
```suggestion
                print("your guess is too high")
```

File: python games/Guessing_game.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                a=int(input("try again:"))
+                if a==0:
+                    print("Thanks for playing!!")
+                    break
+            else:
+                print("Lucky Guess!!")
+                break
+except:
+    print("Told ya!! Enter only Intiger value")
```
Comment (havanagrawal) [inline python games/Guessing_game.py]: *integer
```suggestion
    print("Told ya!! Enter only integer value")
```

File: python games/Guessing_game.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    else:
+        b=random.randint(1,100)
+        while(True):
+            if(b>a):
+                print("your guess is too low")
+                a=int(input("try again:"))
+                if a==0:
+                    print("Thanks for playing!!")
+                    break
+            elif(b<a):
```
Comment (havanagrawal) [inline python games/Guessing_game.py]: Unnecessary brackets, spacing 
```suggestion
            elif b < a:
```

File: python games/Guessing_game.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import random
+print('''Introduction:
+This is guessing game,In this game you have to guess
+any number and after that this will print your guess status,and
+that is indicate your guess is too high or too low from the actual number.
+If your guess is matched to the actual number then you are win!!
+''')
+print('''Instruction:
+    ->Enter 0 for exit the game.
+    ->Enter only intiger value.
```
Comment (havanagrawal) [inline python games/Guessing_game.py]: *integer
```suggestion
    ->Enter only integer value.
```

File: python games/Guessing_game.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    a=int(input("guess one number between 1 to 100:"))
+    if a==0:
+        print("Thanks for playing!!")
+    else:
+        b=random.randint(1,100)
+        while(True):
+            if(b>a):
+                print("your guess is too low")
+                a=int(input("try again:"))
+                if a==0:
```
Comment (havanagrawal) [inline python games/Guessing_game.py]:
```python
if a==0:
    print("Thanks for playing!!")
```

is repeated (duplicated) thrice in the script

File: python games/Guessing_game.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    ->Enter 0 for exit the game.
+    ->Enter only intiger value.
+      ''')
+try:
+    a=int(input("guess one number between 1 to 100:"))
+    if a==0:
+        print("Thanks for playing!!")
+    else:
+        b=random.randint(1,100)
+        while(True):
```
Comment (havanagrawal) [inline python games/Guessing_game.py]:
```suggestion
        while True:
```

File: python games/Guessing_game.py [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+            if(b>a):
```
Comment (havanagrawal) [inline python games/Guessing_game.py]:
```suggestion
            if b > a:
```

**Comment by cclauss** [comment]
See discussion of __input()__ in CONTRIBUTING.md.

**Pull request closed without merging**